src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -551,6 +590,119 @@ public UpdateItemResult updateItem(UpdateItemRequest updateItemRequest) {
         return getAmazonDynamoDb().updateItem(updateItemRequest);
     }
 
+    @Override
+    public ListBackupsResult listBackups(ListBackupsRequest listBackupsRequest) {
+        if (backupManager.isPresent()) {
+            return backupManager.get().listBackups(listBackupsRequest);
+        } else {
+            throw new ContinuousBackupsUnavailableException("Backups can only be created by configuring a backup "
+                + "managed on an mt-dynamo table builder, see <insert link to backup guide>");
+        }
+    }
+
+    @Override
+    public RestoreTableFromBackupResult restoreTableFromBackup(
+        RestoreTableFromBackupRequest restoreTableFromBackupRequest) {
+        if (backupManager.isPresent()) {
+            if (!(restoreTableFromBackupRequest instanceof RestoreMtBackupRequest)) {
```

Comment:
This is the first case where we require a subtype of a request object be passed to the API. Would be good to review whether there are possible alternatives. For example, could we encode whatever additional data we need in the backupArn?